docs/book/src/multiversion-tutorial/testdata/project/config/rbac/kustomization.yaml

```diff
@@ -18,10 +18,11 @@ resources:
 - metrics_auth_role.yaml
 - metrics_auth_role_binding.yaml
 - metrics_reader_role.yaml
-# For each CRD, "Editor" and "Viewer" roles are scaffolded by
+# For each CRD, "Admin", "Editor" and "Viewer" roles are scaffolded by
 # default, aiding admins in cluster management. Those roles are
 # not used by the Project itself. You can comment the following lines
```

Comment:
@damsien could we replace here Project by the project name as well? 
We would need to change the template to use {{ .ProjectName }}